src/Styling/Home.css

```diff
@@ -98,15 +98,15 @@
   position: relative;
 }
 
-.About_image_container img {
+.About_image_border img {
```

Comment:
this parent div should not be targeting the image nested inside it’s child div. The child div (profile_image_container) should be the one targeting and setting the styles for the img